api/lagoon/client/notification.go

```diff
@@ -2,15 +2,34 @@ package client
 
 import (
 	"context"
+	"github.com/machinebox/graphql"
 	"github.com/uselagoon/machinery/api/schema"
 )
 
 // AddNotificationSlack defines a Slack notification.
 func (c *Client) AddNotificationSlack(ctx context.Context, in *schema.AddNotificationSlackInput, out *schema.NotificationSlack) error {
-	req, err := c.newRequest("_lgraphql/notifications/addNotificationSlack.graphql", in)
-	if err != nil {
-		return err
+	var req *graphql.Request
```

Comment:
These could be consolidated down more like this
```
var req *graphql.Request
var err error
variables := map[string]interface{}{
	"name":    in.Name,
	"webhook": in.Webhook,
	"channel": in.Channel,
}
if *in.Organization == 0 {
        variables["organization"] = in.Organization
}
req, err = c.newRequest("_lgraphql/notifications/addNotificationSlack.graphql",variables)
if err != nil {
	return err
}
```

Reply:
Good call! Updated now